lib/kuzzle.js

```diff
@@ -199,6 +199,7 @@ class Kuzzle extends EventEmitter {
         return this.validation.curateSpecification();
       })
       .then(() => this.entryPoints.init())
+      .then(() => this.repositories.role.checkAllRolePluginsRights())
```

Comment:
This check should be handled by the `RoleRepository.init` method which is called by the `Repositories.init` method (and this last method is called in Kuzzle initialization)

Reply:
How I am supposed to do it since this check use the `pluginsManager` which at this state is not initialized ?